src/status_im/ui/screens/events.cljs

```diff
@@ -274,28 +275,29 @@
   :initialize-account
   (fn [_ [_ address events-after]]
     {:dispatch-n (cond-> [[:initialize-account-db address]
-                          [:load-processed-messages]
-                          [:initialize-protocol address]
-                          [:initialize-sync-listener]
-                          [:initialize-chats]
-                          [:load-contacts]
-                          [:load-contact-groups] 
-                          [:init-discoveries]
-                          [:initialize-debugging {:address address}]
-                          [:send-account-update-if-needed]
-                          [:start-requesting-discoveries]
-                          [:remove-old-discoveries!]
-                          [:update-wallet]
-                          [:update-transactions]
-                          [:get-fcm-token]]
+                          ;[:load-processed-messages]
+                          ;[:initialize-protocol address]
+                          ;[:initialize-sync-listener]
+                          ;[:initialize-chats]
+                          ;[:load-contacts]
+                          ;[:load-contact-groups]
+                          ;[:init-discoveries]
+                          ;[:initialize-debugging {:address address}]
+                          ;[:send-account-update-if-needed]
+                          ;[:start-requesting-discoveries]
+                          ;[:remove-old-discoveries!]
+                          ;[:update-wallet]
+                          ;[:update-transactions]
+                          ;[:get-fcm-token]]
+                          ]
                    (seq events-after)
                    (into events-after))}))
 
 (register-handler-fx
   :check-console-chat
   (fn [{{:accounts/keys [accounts] :as db} :db} [_ open-console?]]
     (let [view (if (empty? accounts)
-                 :chat
+                 :login
```

Comment:
@vkjr the place to assign screen id into db's `:view-id` ... later it's loaded in main cljs screen